sandbox/update_yindex_index.json

```diff
@@ -0,0 +1,50 @@
+{
+    "properties": {
+      "module": {
+        "type": "text",
+        "analyzer": "ngram",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "organization": {
+        "type": "text",
+        "analyzer": "ngram",
+        "fields": {
+          "keyword": {
+            "type": "keyword",
+            "ignore_above": 256
+          }
+        }
+      },
+      "argument": {
+        "type": "text",
+        "analyzer": "ngram"
+      },
+      "description": {
+        "type": "text",
+        "analyzer": "description",
+        "fields": {
+          "lowercase": {
+            "type": "text",
+            "analyzer": "description_lowercase"
+          },
+          "keyword": {
+            "type": "keyword"
+          }
+        }
+      },
+      "sdo": {
+        "type": "boolean"
+      },
+      "revision": {
+        "type": "date"
+      },
+      "maturity-level": {
+        "type": "keyword"
```

Comment:
Do we need to store info about maturity-level?
I mean like whether is it not enough to store just some `rfc` flag - similar as for `sdo`
```
      "rfc": {
        "type": "boolean"
      }
```

Reply:
I’m not sure, for this purpose it would be enough but maybe we would have to filter/search for some specific maturity-level in future, as with sdo, we still store the organization and just have another boolean field